Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Merger with Lich King

[**_HOW TO REVIEW LOCALIZATION?_**](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/wiki/How-to-be-Keeper-(Tester)#reviewing-code)
[**_CLICK HERE TO SEE LOCALIZATION_**](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1007/files?file-filters%5B%5D=.csv#diff-fd0a908868d1de03479d4c83134d78a4)

## Changelog:
- Ner'zhul can initiate **Merge with Champion**, an event chain where the holder of Frostmourne and Ner'zhul become one. The event chain ends when the Champion wake ups.

## How to test:
- Start in 603, give Ner'zhul a few years to summon the Legion, switch to Ner'zhul, pick **Merge with Champion** decision, switch to Arthas, see what happens.
- @spacsea and @DukeJeans could you review added localization. I left 2 links for you above. The first link leads to manual (if you need it), the second leads to localization itself.
